apps/back-office/src/app/(dashboard)/settings/team-members/components/AddTeamMember/ServicesForm.tsx

```
@@ -0,0 +1,144 @@
+      if (isSelected) {
+        return prev.filter((s) => s !== service);
+      } else {
+        return [...prev, service];
+      }
+    });
+
+    // Clear error when selection is changed
+    if (errors.services) {
+      setErrors((prev) => {
+        const newErrors = { ...prev };
+        delete newErrors.services;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)